frontend/src/utils/serializeFilterParams.js

```diff
@@ -38,23 +42,25 @@ export default function serializeFilterParams(filters, descriptions) {
 
       case FILTER_TYPE.INPUT: {
         const options = filters[field].options
-
-        if (options) {
-          if (options.recursiveMatch)
-            key += "__recursive"
-          else if (options.exactMatch)
-            key += "__startswith"
-          else
+        const isBatch = description.batch && hasSpaces(value)
+
+        if(!isBatch){
+          if (options) {
+            if (options.recursiveMatch)
```

Comment:
Pure stylistic comment, would there be a way to re-organize a bit the logic to have less nested 'if' statements?